x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/process_hash.tsx

```diff
@@ -20,61 +20,29 @@ const HashFlexGroup = styled(EuiFlexGroup)`
 interface Props {
   contextId: string;
   eventId: string;
-  processHashMd5: string | null | undefined;
-  processHashSha1: string | null | undefined;
   processHashSha256: string | null | undefined;
 }
 
-export const ProcessHash = React.memo<Props>(
-  ({ contextId, eventId, processHashMd5, processHashSha1, processHashSha256 }) => {
-    if (
-      isNillEmptyOrNotFinite(processHashSha256) &&
-      isNillEmptyOrNotFinite(processHashSha1) &&
-      isNillEmptyOrNotFinite(processHashMd5)
-    ) {
-      return null;
-    }
-
-    return (
-      <HashFlexGroup alignItems="center" direction="column" gutterSize="none">
-        {!isNillEmptyOrNotFinite(processHashSha256) && (
-          <TokensFlexItem grow={false} component="div">
-            <DraggableBadge
-              contextId={contextId}
-              eventId={eventId}
-              field="process.hash.sha256"
-              iconType="number"
-              value={processHashSha256}
-            />
-          </TokensFlexItem>
-        )}
-
-        {!isNillEmptyOrNotFinite(processHashSha1) && (
-          <TokensFlexItem grow={false} component="div">
-            <DraggableBadge
-              contextId={contextId}
-              eventId={eventId}
-              field="process.hash.sha1"
-              iconType="number"
-              value={processHashSha1}
-            />
-          </TokensFlexItem>
-        )}
-
-        {!isNillEmptyOrNotFinite(processHashMd5) && (
-          <TokensFlexItem grow={false} component="div">
-            <DraggableBadge
-              contextId={contextId}
-              eventId={eventId}
-              field="process.hash.md5"
-              iconType="number"
-              value={processHashMd5}
-            />
-          </TokensFlexItem>
-        )}
-      </HashFlexGroup>
-    );
+export const ProcessHash = React.memo<Props>(({ contextId, eventId, processHashSha256 }) => {
+  if (isNillEmptyOrNotFinite(processHashSha256)) {
+    return null;
   }
-);
+
+  return (
+    <HashFlexGroup alignItems="center" direction="column" gutterSize="none">
+      {!isNillEmptyOrNotFinite(processHashSha256) && (
```

Comment:
nit -> I do not think that we need this condition `!isNillEmptyOrNotFinite(processHashSha256)` anymore